### PR TITLE
doc: add Notation v1.0.0-RC.5 release blog

### LINKED ISSUE #207: Create a blog post for Notation v1.0.0 RC.5 release
Notation v1.0.0 RC.5 was planned per https://github.com/notaryproject/notation/issues/661. We need to create a blog post accordingly

### PR DESCRIPTION
**Pull request by FeynmanZhou**
Fix #207

### CODE REVIEW

File: content/en/blog/2023/announcing-notation-rc5.md
Comment (yizha1) [inline content/en/blog/2023/announcing-notation-rc5.md:44]: We would like to specially thank the Notation maintainers, contributors, and the broader Notary community for helping us throughout the release process with timely feedback, reviews, community testing and for all your support to help ensure a timely release. Sending credits to the following contributors who made great contributions to RC.5.

File: content/en/blog/2023/announcing-notation-rc5.md
Comment (yizha1) [inline content/en/blog/2023/announcing-notation-rc5.md:40]: Maybe we can specify from RC-4 only.

**Review by yizha1 — COMMENTED** [review]
Thanks @FeynmanZhou, left some comments. I will review it again later.

File: content/en/blog/2023/announcing-notation-rc5.md
Comment (shizhMSFT) [inline content/en/blog/2023/announcing-notation-rc5.md:15]: Be careful with the wording. We DO NOT comply with those new specs but compilable with them.

File: content/en/blog/2023/announcing-notation-rc5.md
Comment (shizhMSFT) [inline content/en/blog/2023/announcing-notation-rc5.md:12]: nit:
```suggestion
Notation is a CLI tool to sign and verify OCI artifacts. The v1.0.0-RC.5 changes include but not limited to the following items. See the [v1.0.0-RC.5 release notes](https://github.com/notaryproject/notation-go/releases/tag/v1.0.0-rc.5) for details.
```

File: content/en/blog/2023/announcing-notation-rc5.md
Comment (shizhMSFT) [inline content/en/blog/2023/announcing-notation-rc5.md:16]: It's SHA256 as SHA2 is a set of hashes such as SHA224, SHA256, SHA384, SHA512, and other variants.
```suggestion
* Use SHA256 instead of insecure SHA1 for certificate thumbprint in `notation inspect`
```

**Review by yizha1 — APPROVED** [review]
LGTM

File: content/en/blog/2023/announcing-notation-rc5.md
Comment (toddysm) [inline content/en/blog/2023/announcing-notation-rc5.md:38]: I am not sure every reader will know what _"Referrers Tag Schema"_ means and what the impact to them is. Can we add more clarification to that.

**Review by toddysm — CHANGES_REQUESTED** [review]

**Review by iamsamirzon — CHANGES_REQUESTED** [review]

**Review by iamsamirzon — CHANGES_REQUESTED** [review]
Let's fix the changes requested by Toddy and I and then we should review again

File: content/en/blog/2023/announcing-notation-rc5.md
Comment (FeynmanZhou) [reply content/en/blog/2023/announcing-notation-rc5.md:29]: I agree with @iamsamirzon that we should not subjectively define whether the RC version of OCI Spec is stable or unstable.  I accepted the suggestions from @iamsamirzon and modified this paragraph accordingly.

File: content/en/blog/2023/announcing-notation-rc5.md
Comment (FeynmanZhou) [reply content/en/blog/2023/announcing-notation-rc5.md]: Agree

File: content/en/blog/2023/announcing-notation-rc5.md
Comment (FeynmanZhou) [reply content/en/blog/2023/announcing-notation-rc5.md:38]: I added a note to explain the concept of the Referrers Tag Schema. See [preview](https://deploy-preview-218--notarydev.netlify.app/blog/2023/announcing-notation-rc5/#change-default-behavior-and-introduce-a-new-flag---allow-referrers-api)

image

**Comment by FeynmanZhou** [comment]
@iamsamirzon @toddysm @yizha1 Thanks for reviewing it. I resolved all comments and accepted most of the comments. Could you review it again?

**Review by yizha1 — DISMISSED** [review]
LGTM

**Review by toddysm — DISMISSED** [review]

File: content/en/blog/2023/announcing-notation-rc5.md
Comment (iamsamirzon) [inline content/en/blog/2023/announcing-notation-rc5.md:10]: Change heading to : Major updates in Notation CLI and notation-go library

File: content/en/blog/2023/announcing-notation-rc5.md
Comment (iamsamirzon) [inline content/en/blog/2023/announcing-notation-rc5.md:14]: Image being referred in what CLI commands, we should give examples  "Notation list", "Notation inspect", Notation Verify.. any others?

File: content/en/blog/2023/announcing-notation-rc5.md
Comment (iamsamirzon) [inline content/en/blog/2023/announcing-notation-rc5.md:15]: Change "add Combability" to "Update compatibility"  I say this because, we had prior compatibility that we just updated to meet the latest revisision

File: content/en/blog/2023/announcing-notation-rc5.md
Comment (iamsamirzon) [inline content/en/blog/2023/announcing-notation-rc5.md:23]: change "before the stable release" to "before v 1.0.0" release.     I say this, because "Stable" is a overloaded term, and I don't want readers to think the prior RC-s were unstable

File: content/en/blog/2023/announcing-notation-rc5.md
Comment (iamsamirzon) [inline content/en/blog/2023/announcing-notation-rc5.md:29]: Again, don't use the words "stable" here. People can misconstrue it.  Just remove the this first sentence ~~"As Notation is close to the first stable release, we are trying to make Notation as stable as possible and avoid breaking changes going forward."~~  and just leave the remaining part as is
Starting with Notation v1.0.0-RC.5 release, Notation will use the [Referrers Tag Schema](https://github.com/opencontainers/distribution-spec/blob/main/spec.md#referrers-tag-schema) as the default behavior since most of the OCI registries are compatible with storing signatures using the Referrers Tag Schema.  The [Referrers API](https://github.com/opencontainers/distribution-spec/blob/main/spec.md#enabling-the-referrers-api) and its fallback strategy are still under development according to recent OCI Distribution Spec changes, so it is marked as experimental in Notation v1.0.0-RC.5.

**Review by iamsamirzon — CHANGES_REQUESTED** [review]

File: content/en/blog/2023/announcing-notation-rc5.md
Comment (FeynmanZhou) [reply content/en/blog/2023/announcing-notation-rc5.md:29]: @iamsamirzon Please note that [Stable](https://en.wikipedia.org/wiki/Software_release_life_cycle#Stable_release) is an official term in Software release life cycle, which means the last release candidate (RC) that has passed all stages of verification and tests. The remaining bugs are considered acceptable. This release goes to [production](https://en.wikipedia.org/wiki/Deployment_environment#Production).

File: content/en/blog/2023/announcing-notation-rc5.md
Comment (FeynmanZhou) [reply content/en/blog/2023/announcing-notation-rc5.md:23]: I changed it to v1.0.0. In fact, [Stable](https://en.wikipedia.org/wiki/Software_release_life_cycle#Stable_release) is an official term in Software release life cycle, which means the last release candidate (RC) that has passed all stages of verification and tests. The remaining bugs are considered acceptable. This release goes to [production](https://en.wikipedia.org/wiki/Deployment_environment#Production).

File: content/en/blog/2023/announcing-notation-rc5.md
Comment (FeynmanZhou) [reply content/en/blog/2023/announcing-notation-rc5.md]: Added two commands

**Review by iamsamirzon — APPROVED** [review]

**Review by iamsamirzon — APPROVED** [review]

**Review by yizha1 — APPROVED** [review]
LGTM

**Pull request merged**
